<ui version="4.0" >
 <class>PutOffForm</class>
 <widget class="QWidget" name="PutOffForm" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>脱扣</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/img/images/cut.png</normaloff>:/img/images/cut.png</iconset>
  </property>
  <property name="styleSheet" >
   <string notr="true" >font: 12pt "Sans Serif";</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>脱扣</string>
   </property>
   <widget class="QComboBox" name="cbNode" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>74</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking" >
     <bool>false</bool>
    </property>
    <property name="autoFillBackground" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>网络号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbNet" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>34</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking" >
     <bool>false</bool>
    </property>
    <property name="autoFillBackground" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>节点号：</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_close" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>关 闭</string>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>确 定</string>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc" />
 </resources>
 <connections/>
</ui>
